website/content/docs/enterprise/lts.mdx

```diff
@@ -102,48 +101,36 @@ consider moving to a Vault Enterprise version with long-term support.
 
 ## What is long-term support?
 
-**Standard support** for Vault Enterprise includes:
+Long-term support offers extended maintenance (minor releases) for select, major
+Vault Enterprise versions.
 
-- performance improvements.
-- feature updates and improvements.
-- bug fixes.
-- security patches.
-- CVE patches to correct vulnerabilities and exposures.
-
-The standard
-[support period and end of life policy](https://go.hashi.co/vault-support-policy)
-covers "N&minus;2" versions, which means, at any given time, HashiCorp supports
+The standard [support period and end of life policy](https://go.hashi.co/vault-support-policy)
+covers "N&minus;2" versions, which means, at any given time, HashiCorp maintains
 the current version ("N") and the two previous versions ("N&minus;2").
 
 Vault versions typically update 3 times per calendar year (CY), which means that
-standard support for a given Vault version lasts approximately 1 year:
-
-Version | Release date         | Standard support ends
-------- | -------------------- | ----------------------
-1.16    | CY24 Q1 (2024-03-27) | CY25 Q1 (1.19 release) 
-1.15    | CY23 Q3 (2023-09-27) | CY24 Q3 (1.18 release) 
-1.14    | CY23 Q2 (2023-06-21) | CY24 Q2 (1.17 release) 
-1.13    | CY23 Q1 (2023-03-01) | CY24 Q1 (2024-03-27) 
-1.12    | CY22 Q4 (2022-10-12) | CY23 Q3 (2023-09-27) 
-
-**Long-term support** extends support for specific Vault Enterprise versions for
-an additional year with:
-
-- patches for bugs that may cause outages.
-- patches for critical vulnerabilities and exposures.
-- high-risk security patches.
-
-After the first year, LTS Vault versions move from standard support to long-term
-support for an additional year.
+**standard maintenance** for a given Vault version lasts approximately 1 year.
+After the first year, LTS Vault versions move from standard maintenance to
+**extended maintenance** for an additional year with patches for bugs that may
+cause outages, critical vulnerabilities and exposures (CVEs), and high-risk
+security patches.
+
```

Comment:
"with patches for bugs that may
cause outages, critical vulnerabilities and exposures (CVEs), and high-risk
security patches."
to 
"with patches for bugs that may cause outages and critical vulnerabilities and exposures (CVEs)."

Reply:
sg. I wasn't sure if we wanted to differentiate between CVEs and "high-risk" security patches.